src/payment/coupon/Benefit.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-i18n';
+
+import messages from './messages';
+import { PERCENTAGE_BENEFIT, ABSOLUTE_BENEFIT } from './data/constants';
+
+const Benefit = (props) => {
```

Comment:
This component displays a description of the applied coupon benefit.

Reply:
Would that be a useful header comment?

Reply:
In general the component names have been pretty self-explanatory, but I think Benefit is less so.